includes/admin/give-metabox-functions.php

```diff
@@ -668,20 +668,20 @@ function give_wysiwyg( $field ) {
 	$field['style']         = isset( $field['style'] ) ? $field['style'] : '';
 	$field['wrapper_class'] = isset( $field['wrapper_class'] ) ? $field['wrapper_class'] : '';
 
-	$field['unique_field_id'] = give_get_field_name( $field );
-	$editor_attributes        = array(
+	$field['unique_field_id'] = str_replace( [ '[', ']' ], [ '_', '' ], give_get_field_name( $field ) );
```

Comment:
@ravinderk Please leave a comment explaining _why_ this is being done.

Reply:
comment added: https://github.com/impress-org/givewp/compare/55ad8ccc71e6...23d8431f0a1f